packages/DataGrid/src/components/Cell/Cell.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as styled from "../../DataGrid.styles";
+
+const propTypes = {
+  a11yText: PropTypes.string.isRequired,
+  column: PropTypes.shape({ cellProps: PropTypes.func, cell: PropTypes.oneOfType([PropTypes.string, PropTypes.func]) })
+    .isRequired,
+  columnIndex: PropTypes.number.isRequired,
+  data: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  gridId: PropTypes.string.isRequired,
+  rowIndex: PropTypes.number.isRequired,
+  style: PropTypes.shape({}).isRequired,
```

Comment:
is this shape correct? Shouldn't it be more specific?

Reply:
Can be any property of the DOM style object in css, I guess it's correct.